### PR TITLE
content/lxd/getting-started-cli.md: add new flag --minimal for lxd init

### LINKED ISSUE #499: [Todo] getting-started-cli.md: Exchange lxd init --auto for newer versions
Related to https://github.com/lxc/lxd/issues/9109

With LXD 4.18 the `lxd init --auto` flag will change to `lxd init --minimal`.

### PR DESCRIPTION
**Pull request by toby63**
Closes #499.

Signed-off-by: Tobias Gerold <tobias@g3ro.eu>

Todo:
- [ ] Wait for 4.0.8 release and change content

### CODE REVIEW

File: content/lxd/getting-started-cli.md [hunk 1]
```diff
@@ -164,7 +164,13 @@ Run the following command to start the interactive configuration process:
 
 **Quick minimal setup**
 
-To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the `--auto` flag:
+To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the following flags:
+
+For LXD 4.18 and newer, use:
+
+    sudo lxd init --minimal
```
Comment (ru-fu) [inline content/lxd/getting-started-cli.md:171]:
```suggestion
To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the ``--minimal`` flag (``--auto`` in LXD versions prior to 4.18):

    sudo lxd init --minimal
```

**Review by ru-fu — COMMENTED** [review]
I think in this kind of getting started guide we can focus on the latest version, to keep it simpler.

File: content/lxd/getting-started-cli.md (hunk 1 above)
Comment (toby63) [reply content/lxd/getting-started-cli.md:171]: 1. I think that the guide should probably cover both the feature release and 4.0.x lts variant. 
2. This is only an intermediant solution, when 4.0.8 is released, we can simply just use `--minimal`.
   So right now I think my version is better readable.

Comment (stgraber) [reply content/lxd/getting-started-cli.md:171]: I'd probably just wait for 4.0.8 to be out and then make the change.

Comment (toby63) [reply content/lxd/getting-started-cli.md:171]: Ok, then I will wait for that.

**Comment by stgraber** [comment]
Let's close for now and re-open once 4.0.8 is out. We'll keep the issue for tracking.

**Pull request closed without merging**